content/docs/started/kubeflow-overview.md

```diff
@@ -0,0 +1,151 @@
++++
+title = "Kubeflow Overview"
+description = "How Kubeflow helps you organize your ML workflow"
+weight = 10
++++
+
+<!--
+Note for authors: The source of the diagrams is held in Google Slides decks,
+in the "Doc diagrams" folder in the public Kubeflow shared drive.
+-->
+
+This guide introduces Kubeflow as a platform for developing and deploying a
+machine learning (ML) system.
+
+Kubeflow is a platform for data scientists who want to build and experiment with
+ML pipelines. Kubeflow is also for ML engineers and operational teams who want
+to deploy ML systems to various environments for development, testing, and
+production-level serving.
+
+## Architectural overview
+
+Kubeflow is *the ML toolkit for Kubernetes*.
+The following diagram shows Kubeflow as a platform for arranging the
+components of your ML system on top of Kubernetes:
+
+<img src="/docs/images/kubeflow-overview-platform-diagram.svg" 
```

Comment:
I left a couple comments on 
https://docs.google.com/presentation/d/1xXu_H9gK5ohNM1y_n2iJFVawurswWiVYS4TSmP1dy3A/edit#slide=id.g70b9dc64b6_0_117

I think combining what's in there with 
https://docs.google.com/presentation/d/1WsrWhLpPzHQ_EkYxVXtIbWbCYbJ6CyyEhf__hlSClus/edit#slide=id.g64cc145417_7_79
might be good.

I think there are two other diagrams that might help here

1. A landscape diagram (https://github.com/kubeflow/website/issues/1450)
2. https://github.com/kubeflow/website/issues/1451

Not suggesting that we should block this PR on those diagrams.

Reply:
Thanks @jlewi and @krishnadurai I've updated the diagram - the updated version will appear in my next commit. I've also added a bit of text below the diagram for added context.

I'll add a summary comment when this PR is ready for review again.

Reply:
Replied on the slide deck.
https://docs.google.com/presentation/d/1xXu_H9gK5ohNM1y_n2iJFVawurswWiVYS4TSmP1dy3A/edit#slide=id.g70b9dc64b6_0_117